core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2398,4 +2398,13 @@ package object config {
       .version("3.3.0")
       .intConf
       .createWithDefault(5)
+
+  private[spark] val EXECUTOR_REMOVE_DELAY =
+    ConfigBuilder("spark.executor.removeDelayOnDisconnection")
```

Comment:
This is specific to standalone right ? Or is it relevant to other resource managers ? (would impact the naming of the config)

Reply:
Yes, this PR only focuses on standalone. Do you have any suggestion about the config name? Maybe `spark.standalone.removeExecutorDelay`? Thank you!

Reply:
How about `spark.standalone.executorRemoveDelayOnDisconnection`?

Reply:
Updated.